apps/backend/src/modules/auth/auth.controller.ts

```diff
@@ -35,6 +35,49 @@ class AuthController extends BaseController {
 				body: userSignUpValidationSchema,
 			},
 		});
+
+		this.addRoute({
+			handler: (options) =>
+				this.getAuthenticatedUser(
+					options as APIHandlerOptions<{
+						// user: { id: number };
+						query: { id: number };
+					}>,
+				),
+			method: "GET",
+			path: AuthApiPath.AUTHENTICATED_USER,
+		});
+	}
+
+	/**
+	 * @swagger
+	 * /auth/authenticated-user:
+	 *   get:
+	 *     description: Return authenticated user
+	 *     security:
+	 *       - bearerAuth: []
+	 *     responses:
+	 *       200:
+	 *         description: Successfull operation
+	 *         content:
+	 *           application/json:
+	 *             schema:
+	 *               type: object
+	 *               $ref: "#/components/schemas/User"
+	 */
+	private async getAuthenticatedUser(
+		options: APIHandlerOptions<{
+			// user: { id: number };
+			query: { id: number };
+		}>,
+	): Promise<APIHandlerResponse> {
+		return {
+			payload: await this.authService.getAuthenticatedUser(
```

Comment:
After the user is injected to the request, you can directly return the user, without fetching it again

Reply:
In my task it says Service (get authenticated user), but I don't see why then, if we can just return the user we got from the token. Am I correct in my understanding that no service is required in this case? Because in order to return the user, it makes no sense to pass it to another method. And do I then need to fix the other suggestions if not using the service (besides ErrorMessage)?

Reply:
The service was mentioned, because it is a part of adding a user to a request. But since that will be done in another ticket, you shouldn't change it

Reply:
So, I don't need to clean up the service, but return the user in it, which we pass, while wrapping it in a try..catch? Or what exactly? I didn't understand

Reply:
What do you mean by "clean up"? You need to work together with Kemal, his task involves getting a user and injecting it to the request, you should get that user from request and send to the client